css/styles.css

```diff
@@ -0,0 +1,111 @@
+body {
+  font-family: Inter, sans-serif;
+  color: white;
+  background-color: #111118;
+  font-weight: bold;
+}
+
+.header {
+  background-color: #1e1e29;
+}
+
+.header > h3 {
+  margin-left: 11px;
+}
+
+img {
+  max-width: 130px;
+}
+
+.content-holder {
+  background-color: #1e1e29;
+  border-radius: 10px;
+}
+
+.margin {
+  margin: 11px 11px 0;
+}
+
+.margin-rigth {
```

Comment:
`rigth` should be `right`

Reply:
<img width="246" alt="Screen Shot 2022-08-02 at 11 38 10 AM" src="https://user-images.githubusercontent.com/106217355/182449006-9314efe2-11a6-41a4-8b6c-0810a4587893.png">